paltas/Configs/config_handler.py

```diff
@@ -240,6 +240,9 @@ def get_lenstronomy_models_kwargs(self,new_sample=True):
 		kwargs_model['source_light_model_list'] = source_model_list
 		kwargs_params['kwargs_source'] = source_kwargs_list
 		kwargs_model['source_redshift_list'] = source_redshift_list
+		kwargs_model['z_source'] = np.max(source_redshift_list)
+		kwargs_model['z_source_convention'] = np.max(source_redshift_list)
```

Comment:
I am just wondering, a z_source_convention must also be chosen when turning the physical mass profiles to lensing deflectors. Currently these settings are chosen at different places, which can cause errors/inconsistencies. Might be best defining the z_source/z_source_conventions more global or at least with one specified routine that then is called multiple times

Reply:
This is a good point. This is now drawn from the z_source parameter required of all source classes (and used by the lens classes as well). This ensures that the same z_source convention is being used everywhere.